tests/test_other.py

```diff
@@ -11349,7 +11349,7 @@ def test_standard_library_mapping(self):
     self.run_process([EMBUILDER, 'build', 'libc-mt-debug', 'libcompiler_rt-mt', 'libdlmalloc-mt'])
 
     libs = ['-lc', '-lcompiler_rt', '-lmalloc']
-    err = self.run_process([EMCC, test_file('hello_world.c'), '-pthread', '-nostdlib', '-v'] + libs, stderr=PIPE).stderr
+    err = self.run_process([EMCC, test_file('hello_world.c'), '-pthread', '-nodefaultlibs', '-v'] + libs, stderr=PIPE).stderr
```

Comment:
Why this change?

Reply:
Because the emscripten_free_tls function is now defined in `crtbegin.o` .. this change means that C runtime libs are skipped but not core startup objects.